sympy/solvers/polysys.py

```diff
@@ -20,6 +20,24 @@ def solve_poly_system(seq, *gens, **args):
     """
     Solve a system of polynomial equations.
 
+    Parameters
+    ==========
+
+    seq: a list/tuple/set
+        Listing all the equations that are needed to be solved
+    *gens: generators
+        generators of the equations in seq for which we want the
+        solutions
+    **args: Keyword arguments
+        Special options for solving the equations
+
+    Returns
+    =======
+
+    A List of tuples
```

Comment:
This should better be `List[Tuple]` or `List` and put the detailed description about the type in the indented block below.

Reply:
@sylee957 You mean something like below:

```
List[Tuple]
    A list of tuples
    Solutions for symbols that satisfy the equations listed
    in polys
```